config/storage-nvme.yml

```diff
@@ -37,86 +37,63 @@ storage:
 # The names of these roles are cross-referenced to groups referred to
 # in playbooks in the ansible/ directory.
 cluster_roles:
-  - name: "ceph_client"
-    groups: "{{ cluster_groups }}"
   - name: "mdadm"
-    groups: [ "{{ storage }}" ]
+    groups: [] #[ "{{ storage }}" ]
   - name: "glusterfs_server"
     groups: [ "{{ storage }}" ]
   - name: "glusterfs_client"
     groups: [ "{{ storage }}" ]
+  - name: "beegfs_mgmt"
+    groups: [ "{{ storage }}" ]
   - name: "beegfs_oss"
     groups: [ "{{ storage }}" ]
+  - name: "beegfs_mds"
+    groups: [ "{{ storage }}" ]
```

Comment:
The inventory generated by this implies we offer both filesystems off the same hardware.  Can that be true?

Reply:
It might be implied as such but cluster_fs argument is the operation code required to make it real. How do you suggest we make it less open to interpretation?